src/containers/Accounts/AccountTransactionsTable/index.js

```diff
@@ -59,11 +59,17 @@ export const AccountTxTable = props => {
     if (newTransactionsRecieved) {
       setState(prevState => ({
         marker: props.data.marker,
-        transactions: prevState.transactions.concat(props.data.transactions),
+        transactions: concatTx(prevState.transactions, props.data.transactions),
       }));
     }
   }, [props]);
 
+  const concatTx = (a, b) => {
+    if (a.length === 0) return b;
+    if (a[0].hash === b[0].hash) return a;
+    return b;
+  };
+
```

Comment:
This method doesn't look like it's actually concatenating anything, what is it supposed to be doing?

Reply:
It's a mistake. I changed it to a function that concatenates.